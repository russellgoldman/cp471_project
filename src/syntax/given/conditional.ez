if (num >= 10) {
} 
elif (num == 5) {
}
