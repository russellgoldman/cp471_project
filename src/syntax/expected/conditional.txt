\Tree [.program [.nextLine [.nextLinePrime [.statement [.ifStatement
	[.if ]
	[.( ]
	[.relationExpression
		[.sumExpression
			[.multiplyExpression
				[.factor [.mutable [.num ] ] ]
				[.None ]
			]
			[.None ]
		]
		[.relationExpressionPrime
			[.relationOperator [.>= ] ]
			[.sumExpression
				[.multiplyExpression
					[.factor [.mutable [.10 ] ] ]
					[.None ]
				]
				[.None ]
			]
			[.None ]
		]
	]
	[.) ]
	[.statementBody
		[.{ ]
		[.None ]
		[.} ]
	]
	[.elifElseStatement
		[.elif ]
		[.( ]
		[.relationExpression
			[.sumExpression
				[.multiplyExpression
					[.factor [.mutable [.num ] ] ]
					[.None ]
				]
				[.None ]
			]
			[.relationExpressionPrime
				[.relationOperator [.== ] ]
				[.sumExpression
					[.multiplyExpression
						[.factor [.mutable [.5 ] ] ]
						[.None ]
					]
					[.None ]
				]
				[.None ]
			]
		]
		[.) ]
		[.statementBody
			[.{ ]
			[.None ]
			[.} ]
		]
		[.None ]
	]
] ] ] ] ]